packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -252,6 +252,47 @@ Future<XcodeBuildResult> buildXcodeProject({
     );
   }
 
+  // If buildNumber is not specified, keep the project untouched.
+  if (buildInfo.buildNumber != null) {
+    final Status buildNumberStatus =
+    logger.startProgress('Running agvtool new-version...', expectSlowOperation: true);
```

Comment:
Indent += 4

Reply:
I don't understand what I did wrong :-(
My change is indented like the unchanged code. What did I did wrong?

Reply:
[Wrapped lines indent an extra four characters](https://www.dartlang.org/guides/language/effective-dart/style#prefer-indenting-continued-lines-with-at-least-four-spaces), so this'd be:

```
    final Status buildNumberStatus =
        logger.startProgress('Running agvtool new-version...', expectSlowOperation: true);
```

Reply:
oooh yes, good catch I didn't saw that